pkg/collector/corechecks/snmp/status/status_templates/snmp.tmpl

```diff
@@ -0,0 +1,29 @@
+{{- if .snmpProfiles }}
+  Profiles
+  ========
+  {{- range $key, $value := .snmpProfiles }}
+  {{$key}}: {{$value}}
+  {{- end }}
+{{- end -}}
+{{- if or .autodiscoverySubnets .discoverySubnets }}
+  Autodiscovery
+  =============
+  {{- range $key, $value := .autodiscoverySubnets }}
```

Comment:
I think we might be able to show more value to users here:
- Each subnet is technically always scanning for refresh although the first scan is arguably the most important
- Knowing where we are in the range would be useful (currently scanning IP 1.2.3.4, 34th IPs of 255) or whatever better text we can use.
- Listing the IPs that were successful, instead of just the number
- In the config, it's common for users to have multiple subnet ranges that are the same, just with different credentials. Because it's a map it might just show one subnet more or less randomly?

Once implemented, can you add a few status examples to the PR description ?